client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/RestService.java

```diff
@@ -420,15 +423,33 @@ public Schema getLatestVersion(String subject)
   }
 
   public Schema getLatestVersion(Map<String, String> requestProperties,
-                                 String subject)
-      throws IOException, RestClientException {
+            String subject)
```

Comment:
Would be good to get rid of the distracting code formatting changes in this method.